doc/script_commands.txt

```diff
@@ -5407,6 +5408,18 @@ Examples:
 
 	// This will end the Freezing status for the invoking character.
 	sc_end SC_FREEZE;
+	
+	//will end the effect of any learned skill for the invoking character.
+	sc_end_class;
+	
+	//will end the effect of any learned skill for the invoking character.
```

Comment:
this is not a sample of an "invoking" character, since you specify a char id of any/another player.
So probably call it "for the specified character" or like you did below say "for the character with the char id 150000.